src/object/fallblock.cpp

```diff
@@ -98,21 +99,21 @@ FallBlock::update(float dt_sec)
 HitResponse
 FallBlock::collision(GameObject& other, const CollisionHit& hit)
 {
-	auto fallblock = dynamic_cast<FallBlock*> (&other);
+  auto fallblock = dynamic_cast<FallBlock*> (&other);
   if (fallblock && hit.bottom && (state == FALL || state == LAND))
   {
-	  physic.set_velocity_y(0.0f);
-	  return CONTINUE;
+    physic.set_velocity_y(0.0f);
+    return CONTINUE;
   }
 
-	auto player = dynamic_cast<Player*>(&other);
-	if (state == IDLE && player && player->get_bbox().get_bottom() < m_col.m_bbox.get_top())
+  auto player = dynamic_cast<Player*>(&other);
+  if (state == IDLE && player && player->get_bbox().get_bottom() < m_col.m_bbox.get_top())
   {
-	  state = SHAKE;
-	  SoundManager::current()->play("sounds/cracking.wav");
-	  timer.start(0.5f);
-	}
-	return FORCE_MOVE;
+    state = SHAKE;
+    SoundManager::current()->play("sounds/cracking.wav");
+    timer.start(0.5f);
+  }
+  return FORCE_MOVE;
 }

```

Comment:
Formatting improvements! :tada:

(Nothing to change, I just want to celebrate the fact that I'm not the only one who cares about it :laughing:)

Reply:
Yeah, mixing tabs and spaces... :rage:

I personally prefer tabs since each coder can set the indentation size to their liking in their editor's preferences without changing the code, but SuperTux uses spaces so I want to see them everywhere. As much as I hate Python, I really want to say `Error: inconsistent use of tabs and spaces` or whatever it was :D

Reply:
I had a reason to prefer spaces over tabs detailed [here](https://github.com/SuperTux/supertux/issues/1552#issuecomment-973093814), but that would be for another time anyways.